documentation/operators/src/legal/landing-pages.md

```diff
@@ -7,3 +7,328 @@ The entire content of this page is under [Creative Commons Attribution 4.0 Inter
+seize this router will accomplish nothing.</p>
+
+<!-- FIXME: US-Only section. Remove if you are a non-US operator -->
+<!--
+<p>
+Furthermore, this machine also serves as a carrier of email, which means that
+its contents are further protected under the ECPA. <a
+href="https://www.law.cornell.edu/uscode/text/18/2707">18
+USC 2707</a> explicitly allows for civil remedies ($1000/account
+<i>plus</i>  legal fees)
+in the event of a seizure executed without good faith or probable cause (it
+should be clear at this point that traffic with an originating IP address of
+FIXME_DNS_NAME should not constitute probable cause to seize the
+machine). Similar considerations exist for 1st amendment content on this
+machine.</p>
+-->
+<!-- FIXME: May or may not be US-only. Some non-US tor nodes have in
+     fact reported DMCA harassment... -->
+<!--
+<p>
+If you are a representative of a company who feels that this router is being
+used to violate the DMCA, please be aware that this machine does not host or
+contain any illegal content. Also be aware that network infrastructure
+maintainers are not liable for the type of content that passes over their
+equipment, in accordance with <a
+href="https://www.law.cornell.edu/uscode/text/17/512">DMCA
+"safe harbor" provisions</a>. In other words, you will have just as much luck
+sending a takedown notice to the Internet backbone providers.
+</p>
+-->
+
+<p>To decentralise and enable privacy for a broad range of services, this
+Exit Gateway adopts an <a href="https://nymtech.net/.wellknown/network-requester/exit-policy.txt">Exit Policy</a>
+in accordance with the <a href="https://tornull.org/">Tor Null ‘deny’ list</a>
+and the <a href="https://tornull.org/tor-reduced-reduced-exit-policy.php">Tor reduced policy</a>,
+which are two established safeguards.
+</p>
+
+<p>
+That being said, if you still have a complaint about the router, you may email the 
+ <a href="mailto:>FIXME>">maintainer</a>. If complaints are related
+ to a particular service that is being abused, the maintainer will submit that to the
+ NYM Operators Community in order to add it to the Exit Policy cited above.
+If approved, that would prevent this router from allowing that traffic to exit through it.
+That can be done only on an IP+destination port basis, however. Common P2P ports are already blocked.</p>
+
+<p>
+You also have the option of blocking this IP address and others on the Nym network if you so desire.
+ The Nym project provides a <a href="https://explorer.nymtech.net/network-components/gateways">
+ web service</a> to fetch a list of all IP addresses of Nym Gateway Exit nodes that allow exiting to a
+specified IP:port combination. Please be considerate when using these options.</p>
+ 
+</main>
+</body>
+</html>
+```
+
+If you're reversing the default 8080 endpoint you may set the landing page accordingly and may resolve it to
+https://nym-exit.yourdomain.org or https://nym-readme.yourdomain.org.
```

Comment:
I would propose to label `yourdomain as` <YOUR_DOMAIN>

Reply:
and adding a little more wording to clarify that